src/importers/cURL/__tests__/Parser-test.js

```diff
@@ -735,16 +778,14 @@ export class TestCurlParser extends UnitTest {
             new Request({
                 url: 'http://httpbin.org/get',
                 method: 'POST',
-                bodyType: 'urlEncoded',
-                body: Immutable.List([
-                    new KeyValue({
-                        key: new FileReference({
-                            filepath: 'filename.txt',
-                            convert: 'stripNewlines'
-                        }),
-                        value: null
-                    })
-                ])
+                bodyType: 'file',
+                body: new FileReference({
+                    filepath: 'filename.txt',
+                    convert: 'stripNewlines'
+                }),
+                headers: Immutable.OrderedMap({
+                    'Content-Type': 'application/x-www-form-urlencoded'
+                })
```

Comment:
My bad! I see the point: the `value` is `null`, so we consider this to be plain body. Great!